docs/download.md

```diff
@@ -43,15 +43,17 @@ The plugin is tested on the following architectures:
 		Apache Spark 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4
 		Apache Spark 3.4.0, 3.4.1, 3.4.2, 3.4.3
 		Apache Spark 3.5.0, 3.5.1
+		Apache Spark 4.0.0
```

Comment:
v24.08.0 release will not contain the 4.0.0 shim, and Apache spark-4.0.0 is not released

Reply:
updated, thx